package.json

```diff
@@ -27,8 +27,11 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint ./src/**/*.tsx",
-    "fix": "eslint ./src/**/*.tsx --fix"
+    "lint": "tslint -p . -c tslint.json && eslint ./src/**/*.tsx",
```

Comment:
Youre right `tslint` is more correct, great fix.

Reply:
I'm getting this when I run it 

```
$ tslint -p . -c tslint.json && eslint ./src/**/*.tsx
Invalid option for configuration: tslint.json
```

Reply:
Wait it looks like it was because I didn't have a tslint.json file. Fixed now.

Reply:
I am confused. Are you making local changes to this branch? That file should be included.